### ISSUE TITLE
JSDoc completions not returned when using @ as trigger character

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/TypeScript#27321]
**TypeScript Version:**  3.1.0-insiders.20180920

**Search Terms:**
- jsdoc
- completion, suggest suggestions
- triggercharacter

**Code**
For the js:
```js
/**
 *
 */
function foo() {}
```

Type `@` inside the jsdoc comment.

**Expected behavior:**
`@` is a trigger character in vscode. This should trigger a request that returns jsdoc tag suggestions 

**Actual behavior:**
Suggestions triggered but no results returned:

```
[Trace  - 1:55:50 PM] Sending request: completionInfo (80). Response expected: yes. Current queue length: 0
Arguments: {
    "file": "/Users/matb/projects/san/b.js",
    "line": 5,
    "offset": 6,
    "includeExternalModuleExports": true,
    "includeInsertTextCompletions": true,
    "triggerCharacter": "@"
}
[Trace  - 1:55:50 PM] Response received: completionInfo (80). Request took 18 ms. Success: false . Message: No content available.
```

**Comment by mjbvz** [comment]
@RyanCavanaugh @andy-ms Let me know if this can be fixed for 3.1. Otherwise I'm going to add a workaround on the vscode side

**Comment by RyanCavanaugh** [comment]
@mjbvz I think it's too late at this point - trying to release Soon (tm)

**Comment by mjbvz** [comment]
Ok, I'm going to stop sending `@` as a trigger character for this version of typescript. Manual completion requests still seem to work

**Comment by mjbvz** [comment]
I added a workaround for TS 3.1 to vscode. Let me know if you have any concerns about us never sending `@` as a trigger character in 3.1

**Comment by nmggithub** [comment]
My issue about this was closed and replaced with this. I'm curious as to why this is being discussed in reference to TypeScript. While I was developing in TypeScript when I noticed this error, it exists in the normal JS language mode as well. Also, I am confused as to what initially caused this error.

**Comment by andy-ms** [comment]
@nmggithub TypeScript is used to check JavaScript files as well.
